package.json

```diff
@@ -32,7 +32,7 @@
     "lint:webext": "web-ext lint -s src --ignore-files='sidebar/vendor/*.js' --self-hosted",
     "package": "npm run build && mv web-ext-artifacts/*.zip addon.xpi",
     "postformat": "npm run lint:js -- --fix",
-    "postinstall": "scripts/postinstall.sh",
+    "postinstall": "bash ./scripts/postinstall.sh",
```

Comment:
sh or bash works either way when I'm using the git bash shell.However, this works only for windows users.Thanks for the update @Natim